lib/openhab/core/actions.rb

```diff
@@ -51,85 +51,12 @@ module Actions
       module_function
```

Comment:
on line 20 of this file, change `NotificationAction` to `{Notification NotificationAction}`, then remove line 22 (and the ending punctuation on line 21)

Reply:
I missed this! Thanks for spotting it!